nanosynth.rb

```diff
@@ -28,11 +28,13 @@
 ###
 ### http://www.joelstrait.com/nanosynth_create_sound_with_ruby/
 
-gem 'wavefile', '=0.7.0'
+gem 'wavefile'
 require 'wavefile'
 
 OUTPUT_FILENAME = "mysound.wav"
 SAMPLE_RATE = 44100
+FORMAT_BITS_PER_SAMPLE = "pcm_16" # Supported values are pcm_8, pcm_16, pcm_24, pcm_32, float_32, and float_64
```

Comment:
Since this is only used to specify the output format, this name could be a little confusing because it implies it is a more general purpose value than it is. Since it is only used in one place, I think it's OK to keep it as a raw value in the code rather than extracting this constant.

Also, this argument is supposed to be a symbol, not a string. I think the WaveFile gem probably happens to work OK if you give the sample format as a string instead of a symbol, but it’s supposed to be a symbol.